actors/builtin/power/power_actor.go

```diff
@@ -533,9 +532,10 @@ func (a Actor) initiateNewSurprisePoStChallenges(rt Runtime) error {
 	rt.State().Transaction(&st, func() interface{} {
 		var err error
 		// sample the actor addresses
-		minerSelectionSeed := rt.GetRandomness(rt.CurrEpoch())
-		randomness := crypto.DeriveRandWithEpoch(crypto.DomainSeparationTag_SurprisePoStSelectMiners, minerSelectionSeed, int64(rt.CurrEpoch()))
+		// minerSelectionSeed := rt.DrawRandomnessFromChain(rt.CurrEpoch())
+		// randomness := crypto.DeriveRandWithMinerAddrAndEpoch(crypto.DomainSeparationTag_SurprisePoStSelectMiners, minerSelectionSeed, int64(rt.CurrEpoch()))
 
+		var randomness []byte
```

Comment:
So what's the goal here ?

Reply:
Sorry this whole section is getting ripped out in https://github.com/filecoin-project/specs-actors/pull/148, so you can ignore all of this